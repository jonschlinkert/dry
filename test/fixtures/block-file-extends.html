{% block "header" %}
header: "block.html"
{% endblock %}
