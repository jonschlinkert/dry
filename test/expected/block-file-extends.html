
unique: "blocks/basic.html"

header: "block.html"
